internal/incoming/web/api_files.go

```diff
@@ -56,6 +68,17 @@ func (c *FilesController) AppendRoutes(router *mux.Router) *mux.Router {
 		Methods("POST").
 		Path("/shards/{shardKey}/files/{fileID}").
 		HandlerFunc(c.CreateFileHandler)
+
+	router.
+		Handle("/.internal/trigger-cutoff", c.fr.TriggerManualCutoff()).
```

Comment:
I'm not sure why we duplicated the endpoint? It exists on the admin server already. We can use our LB to route the existing endpoint if needed.

Reply:
@adamdecaf From what I understand, the idea (according to [this card](https://linear.app/moov/issue/INFRA-643/ach-tooling-trigger-manual-cutoff-and-inbound)) was that since dashboard doesn't have access to the admin service (`9494`), we can add the endpoint to the frontend service (`8484`). Would our LB allow us to route dashboard to these admin endpoints?

Reply:
The LB can route this for us. I was thinking of something like the following:

```yaml
routers: 
    achgateway:
      rule: "Host(`local.moov.io`) && PathPrefix(`/.internal/ach/trigger-cutoff`)"
      service: achgateway-admin
      entryPoints: ['eighty', 'eightyone']
      middlewares:
        # - secured
        - achgateway-internal
        
middlewares:
  achgateway-internal:
      replacePathRegex:
        regex: "/.internal/"
        replacement: ""
```

Reply:
I figured we probably don't want to expose an admin service to the frontend

Reply:
@adamdecaf works for me. I'll drop the first commit, but I'll keep the changes addressing the `gosec` lint errors

Reply:
@adamdecaf , isn't the more proper solution to do what @pa3ng did here and expose the endpoint from the frontend service?  Particularly now that he's done the work for it?  I understand that what you're suggesting is the quicker solution, but it seems dirty to me, especially since we don't do anything like that currently, and the admin router wasn't meant to be exposed to frontend clients

Reply:
achgateway is OSS though, so I don't understand why it needs to duplicate an endpoint to solve an internal Moov problem. The admin endpoints are intended to be hooked up to a dashboard like we're doing, so using the LB to route makes sense.

Reply:
The idea behind admin servers was to allow for these backend operations to be used with dashboards, slackbots, etc. 

Right now achgateway has a "business" [endpoint for creating ACH files](https://github.com/moov-io/achgateway/blob/7dda8b7ebe87bec2de4544eb5345d4890b5ad9cc/internal/incoming/web/api_files.go#L57). This is intended to be used by services to queue up files, but triggering a cutoff/inbound is an admin operation that would typically be behind enhanced permissions.